### PR TITLE
seminars: create tickets for submissions

### PR DESCRIPTION
**Pull request by harunurhan**
* Also sends confirmation email to the user when a new seminar is submitted.

### CODE REVIEW

File: backend/inspirehep/mailing/config.py [hunk 1]
```diff
@@ -15,6 +15,8 @@
 CONFERENCES_CONFIRMATION_EMAIL_ADDRESS = "conf@inspirehep.net"
 """Email address for sending confirmation email for conferences"""
 
+SEMINARS_CONFIRMATION_EMAIL_ADDRESS = "seminars@inspirehep.net"
```
Comment (drjova) [inline backend/inspirehep/mailing/config.py:18]: do we even have this kind of emails?

File: backend/inspirehep/mailing/api/seminars.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import structlog
+from flask import current_app, render_template
+
+from inspirehep.mailing.providers.flask_mail import send_email
+
+LOGGER = structlog.getLogger()
+
+
+def get_submissions_confirmation_email_html_content(seminar):
+    host = current_app.config["SERVER_NAME"]
```
Comment (drjova) [inline backend/inspirehep/mailing/api/seminars.py:17]:
```suggestion
    host = get_inspirehep_url()
```

Comment (harunurhan) [reply backend/inspirehep/mailing/api/seminars.py:17]: full url looks ugly in the email, isn't `https` by default anyhow?

File: backend/inspirehep/mailing/config.py (hunk 1 above)
Comment (harunurhan) [reply backend/inspirehep/mailing/config.py:18]: I am not sure, that was the requirement. but it will work either way right?

Comment (StellaCh) [reply backend/inspirehep/mailing/config.py:18]: see last messages on https://inspirehep.zulipchat.com/#narrow/stream/195298-experts/topic/virtual.20seminars.20in.20INSPIRE

File: backend/inspirehep/mailing/api/seminars.py (hunk 2 above)
Comment (drjova) [reply backend/inspirehep/mailing/api/seminars.py:17]: it's using https https://github.com/inspirehep/inspirehep/blob/b6aa07abab5504eb5744ef74d161a1f5c1e2f208/backend/inspirehep/utils.py#L18

File: backend/inspirehep/mailing/config.py (hunk 1 above)
Comment (drjova) [reply backend/inspirehep/mailing/config.py:18]: 👍

File: backend/inspirehep/mailing/api/seminars.py (hunk 2 above)
Comment (harunurhan) [reply backend/inspirehep/mailing/api/seminars.py:17]: No no I mean, the load balancer.. it's routing to `https` by default and `protocol` seems ugly in the mail content.

**Pull request merged**